source/MRMesh/MRPolylineTrimWithPlane.cpp

```diff
@@ -2,178 +2,276 @@
 #include "MRPolyline.h"
 #include "MRPolylineEdgeIterator.h"
 #include "MRPlane3.h"
+#include "MRGTest.h"
```

Comment:
We decided to put tests not in MRMesh any more. See example here: https://github.com/MeshInspector/MeshLib/pull/4146

Reply:
fixed